app/views/layouts/base.html.erb

```diff
@@ -32,26 +32,26 @@
       <% end %>
     </script>
     <%= javascript_include_tag "locale/#{FastGettext.locale}/app" %>
-    <%= javascript_include_tag *webpack_asset_paths('foreman-vendor', :extension => 'js') %>
-    <%= javascript_include_tag *webpack_asset_paths('vendor', :extension => 'js') %>
-    <%= javascript_include_tag *webpack_asset_paths('bundle', :extension => 'js') %>
-    <%= javascript_include_tag 'application' %>
-    <%= webpacked_plugins_with_global_js %>
-    <%= webpack_dev_server %>
-    <%= yield(:javascripts) %>
-
-    <script type="text/javascript">
-      <%= yield(:inline_javascripts) %>
-    </script>
-
+    
     <%= yield(:head) %>
   </head>
 
   <body class="pf-m-redhat-font">
-    <!--[if lte IE 9]>
+    <!--[if IE]>
```

Comment:
I wanted to look into this. https://learn.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/compatibility/hh801214(v=vs.85) states that IE 10 ignores this so these statements are equal. Given they are equal, I'd personally not make the change but I'll leave it up to you.

Reply:
We are not supporting IE at all, but since IE 10 isn't reading this section, we could keep it as is